docs/en/sql_reference/functions/array_functions.md

```diff
@@ -999,11 +999,15 @@ arrayZip(arr1, arr2, ..., arrN)
 
 **Parameters**
 
-`arr` — Any number of [array](../../sql_reference/data_types/array.md) type columns to combine.
+- `arrN` — [Array](../data_types/array.md).
+
+The function can take any number of arrays. All the input arrays must be of equal size.
 
 **Returned value**
 
-The result of Array\[Tuple(…)\] type after the combination of these arrays
+- Array with elements from the source arrays grouped into [tuples](../data_types/tuple.md).
```

Comment:
We should add that types in tuples are taken from corresponding arrays. No changes in underlying types are performed.